pkg/config/consortiums_test.go

```diff
@@ -113,44 +113,44 @@ func TestAddOrgToConsortium(t *testing.T) {
 									"Admins": {
 										"mod_policy": "Admins",
 										"policy": {
-											"type": 3,
```

Comment:
What's with all of the whitespace changes in here? Seems unrelated to the change.

Reply:
Turns out when I copied the JSON output into the test some whitespaces changed? I'll make sure nothing unnecessary gets committed when updating the expected JSON in the future